src/config/babelrc.js

```diff
@@ -8,11 +8,10 @@ const isWebpack = parseEnv('BUILD_WEBPACK', false)
 const treeshake = parseEnv('BUILD_TREESHAKE', isRollup || isWebpack)
 const alias = parseEnv('BUILD_ALIAS', isPreact ? {react: 'preact'} : null)
 
-const envModules = treeshake ? {modules: false} : {}
 const envTargets = isTest
   ? {node: 'current'}
   : isWebpack || isRollup ? {browsers: ['ie 10', 'ios 7']} : {node: '4.5'}
-const envOptions = Object.assign({}, envModules, {targets: envTargets})
+const envOptions = {modules: false, loose: true, targets: envTargets}
```

Comment:
Hmmm... We only want `{modules: false}` if `treeshake` is true right?

Reply:
im adding transform-es2015-commonjs "by hand" few line below, see [this comment](https://github.com/kentcdodds/kcd-scripts/pull/33#discussion_r166554943)

Reply:
Ah, sorry, I somehow missed that. Looks good!